client/src/i18n/en-US.json

```diff
@@ -916,7 +916,7 @@
         "submit_for_review": "You are about to submit the submission for review.",
         "accept_for_review": "You are about to accept the submission for review.",
         "reject": "You are about to reject the submission for review.",
-        "request_resubmission": "You are about to request the submitter to resubmit this submission as a new submission to be reviewed.",
+        "request_resubmission": "You are about to request resubmission.",
```

Comment:
This edit can remain, but I just wanted to note that we should be keep in mind that the resubmission process in Pilcrow can be improved so that it's easier for users. New users may not understand how resubmission works in Pilcrow and the extra detail here previously was an attempt to better communicate to new users that a submitter must submit a totally separate submission in order to resubmit.

Reply:
The original sentence is grammatically unclear. Is the suggested edit inaccurate?

Reply:
No, it's not inaccurate. As I said, it can remain. I just wanted to detail the thought behind this and make sure we're on the same page.